### PR TITLE
Add docs on how to speed up hashing

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -44,3 +44,16 @@ $ export QUILT_MINIMIZE_STDOUT=true
 1. Go to CloudFormation > Stacks > YourQuiltStack > Outputs
 1. Copy the row labeled TemplateBuildMetadata
 1. "git_revision" is your template version
+
+## Hashing takes a long time. Can I speed it up?
+
+Yes. Follow these steps:
+
+1. Run your compute in the same region as your S3 bucket (as opposed to
+a local machine or foreign region)—I/O is much faster.
```
Comment (sir-sigurd) [inline docs/FAQ.md:53]:
```suggestion
a local machine or foreign region) — I/O is much faster.
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -44,3 +44,16 @@ $ export QUILT_MINIMIZE_STDOUT=true
...
+
+1. Use a larger instance with more vCPUs.
+
+1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](API%20Reference/cli.md#quilt_transfer_max_concurrency)
+above its default of 10 to better
```
Comment (sir-sigurd) [inline docs/FAQ.md:58]: I'd rather not repeat here what value is default.

**Review by sir-sigurd — COMMENTED** [review]
LGTM in general.

**Review by sir-sigurd — APPROVED** [review]

File: docs/FAQ.md (hunk 1 above)
Comment (akarve) [reply docs/FAQ.md:53]: Correct as written and would rather keep style consistent. It looks much better in a variable width font.

File: docs/FAQ.md (hunk 2 above)
Comment (akarve) [inline docs/FAQ.md:58]:
```suggestion
1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](API%20Reference/cli.md#quilt_transfer_max_concurrency)
```

Comment (akarve) [reply docs/FAQ.md:58]: True

Comment (akarve) [inline docs/FAQ.md:58]:
```suggestion
1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](API%20Reference/cli.md#quilt_transfer_max_concurrency)
```

**Pull request merged**